src/databricks/sql/auth/authenticators.py

```diff
@@ -0,0 +1,118 @@
+import base64
+import logging
+from typing import Dict, List
+
+from databricks.sql.auth.oauth import OAuthManager
+
+# Private API: this is an evolving interface and it will change in the future.
+# Please must not depend on it in your applications.
+from databricks.sql.experimental.oauth_persistence import OAuthToken, OAuthPersistence
+
+
+class CredentialsProvider:
```

Comment:
nit: would `AuthProvider` be more consistent with the other names? The implementations all have this as suffix, and the field in which it is stored is also called: `auth_provider`

Reply:
renamed.